tests/components/sensibo/conftest.py

```diff
@@ -43,3 +47,25 @@ async def load_int(hass: HomeAssistant) -> MockConfigEntry:
     await hass.async_block_till_done()
 
     return config_entry
+
+
+@pytest.fixture(name="get_data")
+async def get_data_from_library(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker, load_json: dict[str, Any]
+) -> SensiboData:
+    """Retrieve data from upstream Sensibo library."""
+
+    client = SensiboClient("123467890", aioclient_mock.create_session(hass.loop))
+    with patch("pysensibo.SensiboClient.async_get_devices", return_value=load_json):
+        output = await client.async_get_devices_data()
+    await client._session.close()  # pylint: disable=protected-access
+    return output
+
+
+@pytest.fixture(name="load_json")
+async def load_json_from_fixture() -> SensiboData:
```

Comment:
We don't want to block the event loop.
```suggestion
def load_json_from_fixture() -> SensiboData:
```

Reply:
Consider setting the scope to session for the fixture but take care to copy the data if it needs to be modified in the tests.